src/main/java/org/breedinginsight/brapps/importer/controllers/UploadController.java

```diff
@@ -78,6 +75,10 @@
         } catch (UnprocessableEntityException e) {
             log.info(e.getMessage());
             return HttpResponse.status(HttpStatus.UNPROCESSABLE_ENTITY, e.getMessage());
+        } catch (ValidatorException e) {
+            log.info(e.getMessage());
```

Comment:
Same thing as above for this log message

Reply:
Pushed changes.